src/components/editor/Bindings/index.tsx

```diff
@@ -66,24 +57,30 @@ function BindingsMultiEditor({ readOnly = false }: Props) {
         connectorTag?.resource_spec_schema,
     ]);
 
-    const resourceConfigUpdated = useEvaluateResourceConfigUpdates(
-        editDraftId,
-        resourceConfig
-    );
+    const resourceConfigUpdated = useMemo(() => {
+        let queriedResourceConfig: ResourceConfigDictionary = {};
+
+        draftSpecs[0]?.spec.bindings.forEach((binding: any) => {
+            queriedResourceConfig = {
+                ...queriedResourceConfig,
+                [binding.source]: {
+                    data: binding.resource,
+                    errors: [],
+                },
+            };
+        });
+
+        // TODO (optimization): Evaluate the performance of a hash comparator function.
```

Comment:
Did a quick search: think this probably isn't worth it

https://github.com/divyanshu013/benchmark-object-hash-deep-equal